.github/workflows/flutter-build.yml

```diff
@@ -0,0 +1,27 @@
+name: Flutter CI
+
+on: [push]
+
+defaults:
+  run:
+    working-directory: smart_pet_buddy
+
+jobs:
+  build-sketch:
```

Comment:
Change the jobs name from build-sketch to build-app

Reply:
Good point! Will fix that.

Reply:
@hry625 Updated